lib/system/io.nim

```diff
@@ -186,6 +186,20 @@ proc readChars*(f: File, a: var openArray[char], start, len: Natural): int {.
   ## reads `len` bytes into the buffer `a` starting at `a[start]`. Returns
   ## the actual number of bytes that have been read which may be less than
   ## `len` (if not as many bytes are remaining), but not greater.
+  runnableExamples("-r:off"):
```

Comment:
don't add runnableExamples to a deprecated proc, add it to the non-deprecated overload

Reply:
Okay, my confusion was that it is not deprecated in 1-4 / the current docs.